SECURITY.md

```diff
@@ -1,5 +1,40 @@
 # Security Policy
 
+## Hugging Face Hub, remote artefacts, and remote code
+
+Transformers is an open-source software that has strong ties to the Hugging Face Hub. While it can be used completely
+offline with models downloaded locally, it provides a very simple way to download, use, and manage models locally.
+
+When downloading artefacts that have been uploaded by others on any platform, you expose yourself to risks. Please
+read below for the security recommendations in order to keep your runtime and local environment safe.
+
+### Remote artefacts
+
+Models uploaded on the Hugging Face Hub come in different formats. We heavily recommend uploading and downloading
+models in the [`safetensors`](https://github.com/huggingface/safetensors) format (which is the default prioritized
+by the transformers library), as developed specifically to prevent code execution in your runtime.
+
+Transformers will default to downloading models in this format if available, but will get other formats available
+if `safetensors` isn't available. You can force this format by using the `use_safetensors` parameter.
```

Comment:
```suggestion
To avoid loading pickle files, you should use the `use_safetenstors` parameter. If doing so, in the event that no .safetensors file is present, transformers will error when loading the model.
```